Dockerfile

```diff
@@ -13,5 +13,4 @@ EXPOSE ${PORT}
 
 COPY . /label-studio
 
-RUN pip install -e .
-CMD ["label-studio", "start", "my_project", "--init"]
```

Comment:
Thanks a lot for contributing! It would be nice if you can also update [Docker startup readme] (https://github.com/heartexlabs/label-studio#run-docker)

Reply:
Thank you very much for your kind reviewing!
I agree on removing this line and did it.
```bash
CMD ["label-studio", "start", "my_project", "--init"]
```
But this line should be there because I met some errors missing it.
```bash
RUN pip install -e .
```
Everything else would be fine.